src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PrintDocument.Windows.cs

```diff
@@ -9,7 +9,7 @@ namespace System.Drawing.Printing
     /// <summary>
     /// Defines a reusable object that sends output to the printer.
     /// </summary>
-    [SRDescription(nameof(SR.PrintDocumentDesc))]
+    [DefaultProperty("DocumentName"), DefaultEvent("PrintPage"), SRDescription(nameof(SR.PrintDocumentDesc))]
```

Comment:
@stephentoub do we have a guideline on how the attributes should be included? I mean should each be in one individual line or this is fine?

Reply:
we are not consistent eg
```
   [DefaultProperty("Interval"), DefaultEvent("Elapsed")]
   public partial class Timer : Component, ISupportInitialize
```

Reply:
Ok. Then I think this is fine. Thanks @danmosemsft

Reply:
Pardon, since now this is fine, can we resolve this conversation?